### PR TITLE
Removed error onClickListener to avoid potential leak

### PR DESCRIPTION
**Pull request by leonardo2204**

### CODE REVIEW

**Comment by sockeqwe** [comment]
Looks good so far. Could you please update the Gradle wrapper to `2.14.1.`. It's not building on travis.

What about ViewState support?

File: mvp-common/src/main/java/com/hannesdorfmann/mosby/mvp/sce/MvpSceView.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    @UiThread
+    void showLoading();
+
+    /**
+     * Show the content view.
+     *
+     * <b>The content view must have the id = R.id.formView</b>
+     */
+    @UiThread
+    void showForm();
```
Comment (leonardo2204) [inline mvp-common/src/main/java/com/hannesdorfmann/mosby/mvp/sce/MvpSceView.java]: Hmm good catch... Any ideas ?

File: mvp/src/main/java/com/hannesdorfmann/mosby/mvp/lce/MvpLceFragment.java [hunk 2]
```diff
@@ -164,6 +164,7 @@ protected void animateErrorViewIn() {
     super.onDestroyView();
     loadingView = null;
     contentView = null;
+    errorView.setOnClickListener(null);
```
Comment (leonardo2204) [inline mvp/src/main/java/com/hannesdorfmann/mosby/mvp/lce/MvpLceFragment.java]: This was supposed to be on a separate commit, my bad.

**Comment by leonardo2204** [comment]
Sure, no problem !
I'm working on it, I think by the end of the day I'll have it implemented !

**Comment by leonardo2204** [comment]
I'll drop another PR, I mistakenly mixed it all. 
Sorry about that
